commands/watch.ts

```diff
@@ -181,11 +206,31 @@ exports.builder = yargs => {
     default: false,
   });
 
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addCmsPublishModeOptions(yargs, { write: true });
-  addUseEnvironmentOptions(yargs);
-  addGlobalOptions(yargs);
-
-  return yargs;
+  return yargs as Argv<WatchCommandArgs>;
+}
+
+const builder = makeYargsBuilder<WatchCommandArgs>(
+  watchBuilder,
+  command,
+  describe,
+  {
+    useConfigOptions: true,
+    useAccountOptions: true,
+    useGlobalOptions: true,
+    useEnvironmentOptions: true,
+    useCmsPublishModeOptions: { write: true },
+  }
+);
+
+const watchCommand: YargsCommandModule<unknown, WatchCommandArgs> = {
+  command,
+  describe,
+  handler: handler as unknown as (
```

Comment:
This can just be `handler`. No need for any of the other stuff afaik

Reply:
Whoops, I'll remove it.